common/OpTestOpenBMC.py

```diff
@@ -360,14 +360,15 @@ def hard_reboot(self):
         self.curl.run()
 
     '''
-    power soft server: Not yet implemented (TODO)
-    curl -c cjar b cjar -k -H "Content-Type: application/json" -X POST -d '{"data":
-    []}' https://bmc/org/openbmc/control/chassis0/action/softPowerOff
+    power soft server:
+    curl -c cjar b cjar -k -H "Content-Type: application/json" -X PUT
+    -d '{"data": "xyz.openbmc_project.State.Host.Transition.Off"}'
+    https://bmc/xyz/openbmc_project/state/host0/attr/RequestedHostTransition
     '''
     def power_soft(self):
-        data = '\'{"data" : []}\''
-        obj = "/org/openbmc/control/chassis0/action/softPowerOff"
-        self.curl.feed_data(dbus_object=obj, operation='rw', command="POST", data=data)
+        data = '\'{"data" : \"xyz.openbmc_project.State.Host.Transition.Off\"}\''
+        obj = "xyz/openbmc_project/state/host0/attr/RequestedHostTransition"
+        self.curl.feed_data(dbus_object=obj, operation='rw', command="PUT", data=data)
```

Comment:
It will again resembles the power_off function only. Currently is OpenBMC supported both hard and graceful shutdown?
power_soft-->It should do graceful shutdown. (Equivalent to ipmi power soft)
power_off--->Immediate power off. (Equivalent to ipmi power off).